coretex/networking/requests_manager.py

```diff
@@ -205,6 +206,39 @@ def post(
 
             raise RequestFailedError
 
+    def streamingDownload(
+        self,
+        endpoint: str,
+        destinationPath: Path,
+        ignoreCache: bool,
+        headers: Dict[str, str],
+        parameters: Dict[str, Any]
+    ) -> NetworkResponse:
+
+        with self.__session.get(
+            self.__url(endpoint),
+            stream = True,
+            headers = headers,
+            json = parameters
+        ) as response:
+
+            if destinationPath.is_dir():
+                raise RuntimeError(">> [Coretex] Destination is a directory not a file")
+
+            if destinationPath.exists():
+                if int(response.headers["Content-Length"]) == destinationPath.stat().st_size and not ignoreCache:
+                    return NetworkResponse(response, endpoint)
+
+                destinationPath.unlink(missing_ok = True)
+
+            destinationPath.parent.mkdir(parents = True, exist_ok = True)
```

Comment:
why is the parent being created here? `FolderManager` should already take care of this